sdk/tests/conformance/buffers/buffer-data-and-buffer-sub-data.html

```diff
@@ -94,15 +94,24 @@
 
         { parameter: new ArrayBuffer(0), expectedBufferSize: 0 },
         { parameter: new ArrayBuffer(4), expectedBufferSize: 4 },
+        { parameter: new Uint8Array(new ArrayBuffer(5)), expectedBufferSize: 5 },
+        { parameter: new DataView(new ArrayBuffer(7)), expectedBufferSize: 7 },
 
         { parameter: "WebGL Rocks!", expectedBufferSize: 0 },
         { parameter: { mystring: "WebGL Rocks!" }, expectedBufferSize: 0 },
     ];
 
+    if (SharedArrayBuffer) {
```

Comment:
This needs to be:
`  if (window.SharedArrayBuffer)
`
to avoid throwing an exception in Safari. Could you update here and throughout?